hubble/Makefile

```diff
@@ -11,7 +13,7 @@ TARGET := hubble
 
 all: $(TARGET)
 
-.PHONY: all $(TARGET) release local-release clean
+.PHONY: all $(TARGET) release local-release clean install
```

Comment:
what about install-binary and bash completion targets?

Reply:
I would expect those targets to also be marked `.PHONY`.

IMHO we should define `.PHONY` above each intended target as I think it's much easier to forget to define it when there's one declaration at the top, as opposed to a declaration above most of the targets. Typically when adding new targets, developers will copy an existing example so it'll be easier for less familiar devs to do "the right thing" that way.